.golangci.yml

```diff
@@ -234,7 +234,6 @@ linters:
     - errname # checks that sentinel errors are prefixed with the Err and error types are suffixed with the Error
     - errorlint # finds code that will cause problems with the error wrapping scheme introduced in Go 1.13
     - exhaustive # checks exhaustiveness of enum switch statements
-    - exportloopref # checks for pointers to enclosing loop variables
     - fatcontext # detects nested contexts in loops
```

Comment:
The linters seem outdated. Should we update our config to match the new v1.64.5 version from https://gist.github.com/maratori/47a4d00457a92aa426dbd48a18776322, but keep our custom settings?

Reply:
agree. I did the same for cel-policy on https://github.com/kubewarden/cel-policy/pull/116